### DISCUSSION TITLE
How to compile this on Windows?

### DISCUSSION THREAD

**Discussion by AxilhOriginal** [ChefKissInc/NootRX#46] [Q&A] [answered]
I need to compile this kext in Windows due to I don't have a Mac lying around. Can someone help?

**Comment by holoto** [comment]
use github action build  https://github.com/ChefKissInc/NootRX/blob/master/.github/workflows/main.yml
or 
make a hackintosh to build https://www.reddit.com/r/hackintosh/

**Comment by VisualEhrmanntraut** [comment, marked as answer]
you can't build this without macOS. If you want to use this kext, get it from the latest "CI" GitHub Actions run (https://github.com/ChefKissInc/NootRX/actions/workflows/main.yml)
